OS/DiskOS/Programs/ls.lua

```diff
@@ -10,4 +18,4 @@ for k, f in ipairs(files) do
   color(fs.isFile(path..f) and 7 or 11)
   print(f.." ",false)
 end
-print("")
+print("")
```

Comment:
You still removed the newline at the end of the file, but I would consider this good for merging. At least you fixed most of it (now it's not all red+green hahaha)